angr/knowledge_plugins/variables/variable_manager.py

```diff
@@ -693,7 +702,7 @@ def get_phi_variables(self, block_addr):
             variables[phi] = self._phi_variables[phi]
         return variables
 
-    def input_variables(self, exclude_specials=True):
+    def get_variables_without_writes(self) -> List[SimVariable]:
```

Comment:
Returned list here, but is used as a set elsewhere, Maybe the correct return value should be an OrderedSet?

Reply:
Thanks for the bug report.

Reply:
https://github.com/angr/angr/commit/d8c1b3a0b4c49de014319a96862705b4be348dc9 fixes this issue.